seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/sink/StarRocksSinkWriter.java

```diff
@@ -73,7 +75,7 @@ public void close() throws IOException {
             }
         } catch (IOException e) {
             log.error("Close starRocks manager failed.", e);
-            throw new IOException("Close starRocks manager failed.", e);
+            throw new StarRocksConnectorException(CommonErrorCode.SQL_OPERATION_FAILED, e);
```

Comment:
Close manage failures have nothing to do with SQL, and it is better to define the associated error code.